RCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/append_reservoir_conditions.py

```diff
@@ -0,0 +1,34 @@
+# RCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/append_reservoir_conditions.py
+# 
+# Created:  Aug 2024, S. Shekar
+
+# ---------------------------------------------------------------------------------------------------------------------- 
+#  IMPORT
+# ---------------------------------------------------------------------------------------------------------------------- 
+import RCAIDE
+from RCAIDE.Framework.Mission.Common     import   Conditions
+
+# ---------------------------------------------------------------------------------------------------------------------- 
+#  append_reservoir_conditions
+# ---------------------------------------------------------------------------------------------------------------------- 
+def append_reservoir_conditions(reservoir,segment,coolant_line,add_additional_network_equation):
+    
```

Comment:
Function headers

Reply:
None of the append_initial_conditions have any function headers, so I haven't included them to keep it consistent.

Reply:
Done